### PR TITLE
feat: add short link to PRs

### PR DESCRIPTION
**Pull request by ansgarm**

### CODE REVIEW

File: cdk.tf/vercel.json [hunk 1]
```diff
@@ -33,6 +33,11 @@
       "destination": "https://github.com/hashicorp/terraform-cdk/issues/:id",
       "permanent": false
     },
+    {
+      "source": "/prs/:id",
```
Comment (mutahhir) [inline cdk.tf/vercel.json:37]: Since this is a team-only thing, let's put this under an `internal` or such path.

Comment (ansgarm) [reply cdk.tf/vercel.json:37]: But that makes the url longer 😄
(And we already have /issues; see directly above)

**Review by DanielMSchmidt — APPROVED** [review]

File: cdk.tf/vercel.json (hunk 1 above)
Comment (xiehan) [reply cdk.tf/vercel.json:37]: Couldn't you technically reuse that one? All PRs are issues (but not all issues are PRs).
https://cdk.tf/issues/2540

Comment (ansgarm) [reply cdk.tf/vercel.json:37]: Right.. Yeah, I could do that 💡

**Pull request closed without merging**